### PR TITLE
LOC CHECKIN | dotnet/wpf master | 20190618

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

**Comment by vatsan-madhavan** [comment]
@msftbot, require signoff from @ryalanms

**Comment by vatsan-madhavan** [comment]
@ryalanms , msftbot did something strange here and merged the PR before the rules were satisfied. Would you mind taking a look to ensure all is well with this PR?

File: src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.es.xlf [hunk 1]
```diff
@@ -74,12 +74,12 @@
       </trans-unit>
       <trans-unit id="LocalizedControlTypeHyperlink">
         <source>hyperlink</source>
-        <target state="translated">hipervínculo</target>
+        <target state="translated">hyperlink</target>
```
Comment (ryalanms) [inline src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.es.xlf]: LocalizedControlType* strings are used to register control types for UI Automation.  These strings should be localized.

File: src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.fr.xlf [hunk 2]
```diff
@@ -74,7 +74,7 @@
       </trans-unit>
       <trans-unit id="LocalizedControlTypeHyperlink">
         <source>hyperlink</source>
-        <target state="translated">lien hypertexte</target>
+        <target state="translated">hyperlink</target>
```
Comment (ryalanms) [inline src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.fr.xlf]: All LocalizedControlType* strings should remain localized for all languages.  Please review LocalizedControlTypeTable as well.

File: src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.it.xlf [hunk 3]
```diff
@@ -194,7 +194,7 @@
       </trans-unit>
       <trans-unit id="LocalizedControlTypeTreeView">
         <source>tree view</source>
-        <target state="translated">visualizzazione albero</target>
+        <target state="translated">tree view</target>
```
Comment (ryalanms) [inline src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.it.xlf]: And here...

File: src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.ja.xlf [hunk 4]
```diff
@@ -74,17 +74,17 @@
...
       <trans-unit id="LocalizedControlTypeHyperlink">
         <source>hyperlink</source>
-        <target state="translated">ハイパーリンク</target>
+        <target state="translated">hyperlink</target>
         <note />
       </trans-unit>
       <trans-unit id="LocalizedControlTypeImage">
         <source>image</source>
-        <target state="translated">イメージ</target>
+        <target state="translated">image</target>
```
Comment (ryalanms) [inline src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.ja.xlf]: And here...

File: src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.zh-Hans.xlf [hunk 5]
```diff
@@ -74,12 +74,12 @@
       </trans-unit>
       <trans-unit id="LocalizedControlTypeHyperlink">
         <source>hyperlink</source>
-        <target state="translated">超链接</target>
+        <target state="translated">hyperlink</target>
```
Comment (ryalanms) [inline src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.zh-Hans.xlf]: The three that I see are hyperlink, image, and table.  These should remain localized.  Thanks.

**Pull request merged**